internal/handler/queue.go

```diff
@@ -20,11 +20,16 @@ const queueId key = 0
 
 func GetQueue(w http.ResponseWriter, r *http.Request) {
 	id := r.Context().Value(queueId).(string)
+	uid := r.Context().Value("uid").(string)
+	log.Println(uid)
```

Comment:
nit: Remove the println

Reply:
Just added that to avoid unused error for now.
Once I use uid in the handler, will remove the print statment.